include/m_string.h

```diff
@@ -28,15 +28,6 @@
 
 #include <string.h>
 
-<<<<<<< HEAD
```

Comment:
Is it OK? For some reason, PS deleted this part.

Reply:
This is Ok. This was our fix for OSx build issues. Upstream now fixed same thing in a different way so we can just use their version.